### DISCUSSION TITLE
Audi A5 2018 - EU Maps won't update

### DISCUSSION THREAD

**Discussion by Mr-Chewie** [Mr-MIBonk/M.I.B._More-Incredible-Bash#186] [Q&A] [answered]
Hi there,

I have installed MIB with no problems and run the AIO and have 023000ee showing as legal in the REM. 

I currently have 2021 maps installed for the UK (where I live) and am trying to update to the latest (2023) version. I have downloaded the latest maps from the Volkswagen download site and extracted to a USB stick.
Updating via the User Interface gives the following error:
![UI Update 3](https://user-images.githubusercontent.com/131666252/233987827-5fa211f0-848e-48d8-ae21-2bce4aba1236.jpg)
Updating via the REM gives the following error:
![REM Update Error](https://user-images.githubusercontent.com/131666252/233987100-540fd991-f842-41b7-a444-95d978e1d8af.jpg)
In the REM my FEC shows as follows:
![REM FEC List](https://user-images.githubusercontent.com/131666252/233987149-82ae4201-9cc4-4dc2-ad03-7434c0ccfec3.jpg)
And in the MIB GEM the Nav settings are as follows:
![MIB GEM Nav Info](https://user-images.githubusercontent.com/131666252/233987191-28a21c79-19b3-4976-a1a2-e9607a9ca4dd.jpg)

I am hoping I have missed a simple step but I cannot see it for looking at present, any advice would be greatly appreciated.
I have included my MIB log below for info also:

[MU1177-JTB-07409.01.1811760595.log](https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/files/11310052/MU1177-JTB-07409.01.1811760595.log)

Thanks in advance :)

**Comment by harman-f** [comment]
Something starnge with Speech files. Have not seen this error before.
Coding looks fine.

Was unit patched before?

**Reply by Mr-Chewie** [reply]
Not as far as I am aware. I got the car in February but get the impression it hadn’t been touched prior to me.

**Reply by Mr-Chewie** [reply]
Any idea on the best way forward to get past the error? Could it be the update info on the USB stick that is to blame? Is there anything I can check on that?

**Reply by harman-f** [reply]
Not sure why this should cause that error. But which format is on your usb stick?
Most likely not fat32

**Reply by Mr-Chewie** [reply]
Just checked and it is FAT32.
I'll try re-downloading the nav update from the VW server and copying onto the stick after re-formatting again and will let you know if that makes a difference.
Is there any other logs or checks that can be done to check the nav update to see if it is corrupt that you are aware of?

thanks
Paul

**Reply by Mr-Chewie** [reply]
In fact I think I have found what could be the problem, each of the folders in the SpeechResVDE folder have three files (content.pkg, Country.iso and a hashes.txt) except for Great Britain which is missing the hashes.txt file.
I will re-copy to the stick and check tat file appears and update.

**Reply by harman-f** [reply]
Was your issues fixed by a fresh SD card?

**Reply by Mr-Chewie** [reply]
Going to try updating on the way to work tomorrow. Will let you know how I get on once I get to work

**Reply by Mr-Chewie** [reply]
Well tried this morning and got a different error. I'm thinking it's an issue with the USB stick I'm using or it just not transferring properly. I have a new 64Gb SD Card that's just arrived so will try re-downloading and extracting to that and see how it goes!

**Comment by Mr-Chewie** [comment, marked as answer]
Well,  firstly apologies for the lack of follow up on this.
I re-downloaded the VW nav package and extracted to a new 64Gb SD card.
Inserted into SD2 slot and started the update....worked a charm.
So issues was either the USB stick I was using or just the copy/extraction process missed out a few files.
